modules/compute-vm/main.tf

```diff
@@ -172,12 +193,26 @@ resource "google_compute_instance" "default" {
   }
 
   boot_disk {
-    auto_delete             = var.boot_disk.auto_delete
-    source                  = var.boot_disk.source
-    disk_encryption_key_raw = var.encryption != null ? var.encryption.disk_encryption_key_raw : null
-    kms_key_self_link       = var.encryption != null ? var.encryption.kms_key_self_link : null
+    auto_delete = var.boot_disk.auto_delete
```

Comment:
I think here we can always have auto_delete = false if it is an independent_disk.